### DISCUSSION TITLE
Unable to select custom time to view the runtime logs

### DISCUSSION THREAD

**Discussion by pablo-k-neruda** [vercel/community#6225] [Help] [answered]
### Page to Investigate

projects->deplyment> runtime logs

### Steps to Reproduce

What ever I select as start and end time to filter runtime logs, the selected time range remains the last hour. Please see the screenshot attached. Hitting enter key does not close the calendar and clicking outside otf the calendar or hitting escape key, closes the calendar but does not update the time.
![image](https://github.com/vercel/community/assets/150379378/8ea07b18-e31c-4220-9e28-0bf8b3351a61)

**Comment by amyegan** [comment, marked as answer]
Hey @pablo-k-neruda. If you're on a Hobby plans, this makes sense as the Hobby limit for logs is currently 1 hour.

https://vercel.com/docs/observability/runtime-logs#limits
